inductiva/simulation/command.py

```diff
@@ -0,0 +1,19 @@
+"""Class for construction of simulator commands."""
+
+
+class Command():
+    """Base class for construction of simulator commands."""
+
+    def __init__(self, **kwargs):
+        """Command constructor.
+        Args:
+            method_name: The name of the method to be executed in 
+            the simulation API.
```

Comment:
I think you should keep `method_name` in this class, but if you don't please remember to remove this.

Reply:
I also think it should be named `name` and not `method_name`. Introducing the `method` prefix is confusing. This is the name of the command, after all...